### PR TITLE
papirus-icon-theme: init at 20170616

### PR DESCRIPTION
**Pull request by bugworm**
###### Motivation for this change
Wanted to add papirus icon theme

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by bugworm** [comment]
```
output path ‘/nix/store/4hk541byh3jyd2vrcdy6cf0xjv26xbkn-papirus-icon-theme-9be3f0f688f33d538f135b590579466946d0bf4c-src’ has r:sha256 hash ‘1daa1izrhc682vx21pwry8fjmr1f7kff7ycdppv362hrjf2hnj28’ when ‘008nkmxp3f9qqljif3v9ns3a8mflzffv2mm5zgjng9pmdl5x70j4’ was expected
```
And it fails only on 1 test, wut?

**Comment by Mic92** [comment]
Please use `git rebase` in future instead of `git pull` if you update your branch. This makes review easier.

**Comment by Mic92** [comment]
I suspect yet another case of unicode normalisation of the filesystem on OS X.

File: pkgs/data/icons/papirus-icon-theme/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        rev = "9be3f0f688f33d538f135b590579466946d0bf4c";
+        sha256 = "008nkmxp3f9qqljif3v9ns3a8mflzffv2mm5zgjng9pmdl5x70j4";
+    };
+
+    dontBuild = true;
+
+    installPhase = ''
+        install -dm 755 $out/share/icons
+        cp -dr --no-preserve='ownership' Papirus{,-Dark,-Light} $out/share/icons/
+        cp -dr --no-preserve='ownership' ePapirus $out/share/icons/
```
Comment (Mic92) [inline pkgs/data/icons/papirus-icon-theme/default.nix]: ownership always ends up root in nix store.

File: pkgs/data/icons/papirus-icon-theme/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+    name = "papirus-icon-theme-${version}";
+    version = "20170616";
+
+    src = fetchFromGitHub {
+        owner = "PapirusDevelopmentTeam";
+        repo = "papirus-icon-theme";
+        rev = "9be3f0f688f33d538f135b590579466946d0bf4c";
```
Comment (Mic92) [inline pkgs/data/icons/papirus-icon-theme/default.nix]: variable should be reused here to make updating easier.

```
  rev = "${version}";
```

File: pkgs/data/icons/papirus-icon-theme/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+    name = "papirus-icon-theme-${version}";
```
Comment (Mic92) [inline pkgs/data/icons/papirus-icon-theme/default.nix]: We use 2 spaces to indent. See: https://github.com/NixOS/nixpkgs/blob/master/.editorconfig

**Review by Mic92 — COMMENTED** [review]
I added same remarks. I already applied the changes and committed directly to master.

**Pull request closed without merging**